ddtrace/tracer/option.go

```diff
@@ -540,6 +544,7 @@ func WithDebugMode(enabled bool) StartOption {
 func WithLambdaMode(enabled bool) StartOption {
 	return func(c *config) {
 		c.logToStdout = enabled
+		c.sendRetries = 2
```

Comment:
Enabling this here will not do anything, since `logToStdout` causes the tracer to use the `logTraceWriter` instead of the `agentTraceWriter`.

Reply:
Sending data to stdout is now optional. By default we now use the regular `agentTraceWriter`. See https://github.com/DataDog/datadog-lambda-go/blob/6b73b08221845f90542e1ea7539f27da26715c6e/internal/trace/listener.go#L72.

Reply:
I see now that this can still work if you specify `WithLambdaMode(false)`, but to me this is confusing.

Another option would be more appropriate I think.